packages/discord.js/src/structures/Message.js

```diff
@@ -417,6 +417,28 @@ class Message extends Base {
     } else {
       this.poll ??= null;
     }
+
+    /**
+     * The call associated with the message
+     * @typedef {Object} MessageCall
+     * @property {?Date} endedAt The time the call ended
+     * @property {?number} endedTimestamp The timestamp the call ended
+     * @property {Snowflake[]} participants The ids of the users that participated in the call
+     */
+
+    if (data.call) {
+      /**
+       * The call associated with the message
+       * @type {?MessageCall}
+       */
+      this.call = {
+        endedAt: data.call.ended_timestamp && new Date(data.call.ended_timestamp),
```

Comment:
This should work, but test it first!

```suggestion
        get endedAt() {
        	return this.endedTimestamp && new Date(this.endedTimestamp)
        },
```